Mathlib/Topology/LocallyConstant/Basic.lean

```diff
@@ -610,11 +610,13 @@ section Piecewise
 
 /-- Given two closed sets covering a topological space, and locally constant maps on these two sets,
     then if these two locally constant maps agree on the intersection, we get a piecewise defined
-    locally constant map on the whole space. -/
+    locally constant map on the whole space.
+
+TODO: Generalise this construction to `ContinuousMap`. -/
 def piecewise {C₁ C₂ : Set X} (h₁ : IsClosed C₁) (h₂ : IsClosed C₂) (h : C₁ ∪ C₂ = Set.univ)
```

Comment:
You can maybe make `f` and `g` implicit (but it depends on how this is used).

Reply:
It's more convenient to have them explicit where I use this in Nöbeling, so I'll leave it as is for now.